resolwe_bio/descriptors/sample2.yml

```diff
@@ -0,0 +1,387 @@
+            If your sample was treated with a chemical agent search for the CHEBI number at
+            https://www.ebi.ac.uk/chebi/, e.g. cisplatin, CHEBI:27899.
+          required: false
+        - name: treatment_protocol
+          label: Treatment protocol
+          type: basic:string
+          description: >
+            Describe how your sample was treated, at different timepoints, if using non-chemical
+            treatments e.g. exercise, aging, etc.
+          required: false
+        - name: annotator
+          label: Annotator
+          type: basic:string
+          required: false
+          description: Person who annotated the sample.
+    - name: sample_processing
+      label: Sample processing
+      group:
+        - name: assay_type
+          label: Assay type
+          type: basic:string
+          required: false
+          choices:
+            - label: RNA-seq
+              value: rna-seq
+            - label: ChIP-seq
+              value: chip-seq
+            - label: ATAC-seq
+              value: atac-seq
+            - label: ChIPmentation
+              value: chipmentation
+            - label: Other
+              value: other
+        - name: extract_protocol
+          label: Extract protocol
+          type: basic:string
+          description: >
+            Describe the protocols used to extract and prepare the material to be sequenced.
+          required: false
+        - name: molecule
+          label: Library made from / Molecule
+          type: basic:string
+          description: >
+            Select what molecule your samples are from, e.g. totalRNA, polyA RNA, genomic DNA, etc.
+          required: false
+          choices:
+            - label: Total RNA
+              value: total_rna
+            - label: polyA RNA
+              value: polya_rna
+            - label: Cytoplasmic RNA
+              value: cytoplasmic_rna
+            - label: Nuclear RNA
+              value: nuclear_rna
+            - label: Genomic DNA
+              value: genomic_dna
+            - label: Protein
+              value: protein
+            - label: Other
+              value: other
+        - name: library_depleted_in
+          label: Library depleted in
+          type: basic:string
+          description: >
+            Select how your library was depleted, otherwise choose N/A.
+          required: false
+          choices:
+            - label: rRNA
+              value: rrna
+            - label: polyA mRNA
+              value: polya_mrna
+            - label: N/A
+              value: na
+        - name: library_treatment
+          label: Library treatment
+          type: basic:string
+          description: Describe how your library was treated.
+          required: false
+        - name: replication_type
+          label: Replication type
+          type: basic:string
+          description: >
+            Select what replication type you used, for more info, go to
```

Comment:
Use hyperlinks

Reply:
We don't support hyperlinks in description of inputs.